Sources/Orbit/Components/Tabs.swift

```diff
@@ -0,0 +1,255 @@
+import SwiftUI
+
+public enum TabsDistribution {
+    /// Distributes tab widths equally (potentially make some tabs having multiline text, if enabled).
+    case equal
+    /// Distributes tab widths based on each tab intrinsic size.
+    case intrinsic
+}
+
+/// Separates content into groups within a single context.
+///
+/// The following example shows component with 3 tabs:
+///
+///     var body: some View {
+///         Tabs {
+///             Tab("one")
+///             Tab("two", style: .underlinedGradient(.ink))
+///             Tab("three", style: .underlined(.blueNormal))
+///         }
+///     }
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/structure/tabs/)
+public struct Tabs<Content: View>: View {
+
+    public let radius: CGFloat = 6
+    public let underlineHeight: CGFloat = 2
+
+    @Binding var selectedIndex: Int
+    let lineLimit: Int?
+    let distribution: TabsDistribution
+    let content: () -> Content
+
+    public var body: some View {
+        HStack(alignment: .center, spacing: 0) {
+            content()
+                .lineLimit(lineLimit)
+                .frame(maxWidth: maxTabWidth, minHeight: Layout.preferredSmallButtonHeight, maxHeight: .infinity)
+        }
+        .fixedSize(horizontal: false, vertical: true)
+        .opacity(0)
+        .backgroundPreferenceValue(Tab.PreferenceKey.self) { preferences in
+            GeometryReader { geometry in
+                tabs(for: preferences, in: geometry)
+            }
+        }
+        .background(background)
+    }
+
+    @ViewBuilder var background: some View {
+        RoundedRectangle(cornerRadius: radius)
+            .fill(Color.cloudLight)
+            .overlay(
+                RoundedRectangle(cornerRadius: radius)
+                    .stroke(Color.cloudNormal, lineWidth: BorderWidth.thin)
+            )
+    }
+
+    @ViewBuilder func tabs(for preferences: Tab.PreferenceKey.Value, in geometry: GeometryProxy) -> some View {
+        HStack(alignment: .center, spacing: 0) {
+            ForEach(preferences.indices, id: \.self) { index in
+                tab(
+                    index,
+                    lastIndex: preferences.indices.count - 1,
```

Comment:
`indices` is not necessary.

You could probably alternatively spell this `preferences.endIndex - 1` to avoid calling `count`, but it probably doesn't matter.